File: C#-Original/Model/Rental.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SharingTm.Model
{
    public class Rental
    {
        public int rentalID { get; set; }
        public int userID { get; set; }
        public string userName { get; set; } // 출력용
        public int timeMachineID { get; set; }
        public int departStationID { get; set; }
        public string departStationName { get; set; } // 출력용
        public DateTime rentalTime { get; set; }
        public int arriveStationID { get; set; }
        public string arriveStationName { get; set; } // 출력용
        public DateTime returnTime { get; set; }
        public string destinationPoint { get; set; }
    }
}

File: Python-Gemini/models/rental.py
from dataclasses import dataclass
from datetime import datetime
from typing import Optional

@dataclass
class Rental:
    """대여 정보를 담는 데이터 클래스"""
    rentalid: int
    userid: int
    username: str
    tmid: int
    depart_rs: int
    depart_station_name: str
    rental_time: datetime
    arrive_rs: int
    arrive_station_name: str
    return_time: Optional[datetime]
    destination_point: str